opencog/util/oc_omp.h

```diff
@@ -44,13 +44,16 @@
 ///@{
 
 //! compile with multithread support (!)
+#ifndef CYGWIN
 #define OC_OMP
+#endif
```

Comment:
I still don't understand why that is needed. I vaguely recall being able to run MOSES in parallel under Windows, the only difference would be that it was slower because thread creation under Windows is slower. But I might have imagined it, I'm not sure.